satpy/tests/reader_tests/test_modis_l1b.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Satpy developers
```

Comment:
```suggestion
# Copyright (c) 2019-2021 Satpy developers
```

Reply:
This should probably be just 2021. This file is new.